packages/synapse-react-client/src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -51,12 +51,6 @@
     padding: 10px 24px;
   }
 
-  // Extra-wide button
-  .btn.btn-wide {
-    min-width: 168px;
-    padding: 10px;
-  }
```

Comment:
Remove class -- converted all bootstrap buttons using 'btn-wide' to MUI styled WideButton components